src/test/java/org/example/hsql/MigrateTablesFromH2Test.java

```diff
@@ -0,0 +1,81 @@
+package org.example.hsql;
```

Comment:
The only reason, I would recommend leaving it in there is that it provides a template that other people could re-use to put together their own tests.  I agree that a developer would want to create their own tests, but this would give them a starting point.  Otherwise they have to go review the MSB project.  

But I understand your point and I would be ok taking it out as well.